guides/common/modules/proc_upgrading-the-external-database.adoc

```diff
@@ -1,25 +1,18 @@
-[id="Upgrading_the_External_Database_{context}"]
+[id="Upgrading_the_External_Database{context}"]
 = Upgrading the external database
 
-You can upgrade an external database from {EL} 8 to {EL} 9 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+The external database will be upgraded when running the {foreman-installer} on the {Project} server.
```

Comment:
```suggestion
If your {Project} deployment uses an external database, the database will be upgraded when running the {foreman-installer} on the {Project} server.
```
Similar as above, just to give an extra hint?

Reply:
I think this helps :+1: